app/views/pages/home.html.erb

```diff
@@ -63,8 +63,8 @@
     <div class="home-page-feature-section-container-alt section-fade-in">
       <div class="home-page-feature-section-content-container-alt">
         <h2 class="home-page-feature-section-supertext">Charts</h2>
-        <p class="home-page-feature-section-header">Create simple, one-click visualisations</p>
-        <p class="home-page-feature-section-paragraph">Semper curabitur ullamcorper posuere nunc sed. Ornare iaculis bibendum malesuada faucibus lacinia porttitor. Pulvinar laoreet sagittis viverra duis. In venenatis sem arcu pretium pharetra at. Lectus viverra dui tellus ornare pharetra.</p>
+        <p class="home-page-feature-section-header">Simple, one-click visualisations</p>
+        <p class="home-page-feature-section-paragraph">Pythia dynamically generates graphics for the data you've chosen to investigate. Export clean, beautifully-designed visualisations to use in reports and slide decks.</p>
```

Comment:
the image for this bit is the same as the image above it, both of the query form im not sure if thats intended or if this parts supposed to have a chart image